activities/member-relations/user-mapping/index.md

```diff
@@ -0,0 +1,29 @@
+---
+type: Resource
+---
+
+# Value proposition hypotheses
+
+> This information is incomplete - it reflects our progress at our first session (April 2019).
```

Comment:
I'd say all of our progress is incomplete, and will likely always be. I feel completeness is not something we should, in its own right, try to achieve. If you would like to front-load that this is not up to par yet, please use different wording. However I feel we'd do better to see this as 'done' since it has direct value, and show our dedication by adding to it.